src/workloads/execution/AggregateExpressions.yml

```diff
@@ -0,0 +1,649 @@
+
+- ExpressionExp:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: ExpressionExp
+      active: [15]
+      operationMetricsName: AggExpressionExp
+      pipeline: [{$project: {exp: {$exp: "$int"}}}]
+
+- ExpressionFilter:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: ExpressionFilter
+      active: [16]
+      operationMetricsName: AggExpressionFilter
+      pipeline: [{$project: {filter: { $filter: {input: "$array", as: "a", cond: { $eq: ["$$a.b", 0] }}}}}]
+
+- ExpressionFloor:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: ExpressionFloor
+      active: [17]
+      operationMetricsName: AggExpressionFloor
+      pipeline: [{$project: {floor: { $floor: "$double"}}}]
+
+- ExpressionIfNull:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: ExpressionIfNull
+      active: [18]
+      operationMetricsName: AggExpressionIfNull
+      pipeline: [{$project: {ifNull: { $ifNull: ["$nullVal", 0]}}}]
+
+- ExpressionIndexOfBytes:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: ExpressionIndexOfBytes
+      active: [19]
+      operationMetricsName: AggExpressionIndexOfBytes
+      pipeline: [{$project: {indexOfBytes: { $indexOfBytes: ["$hello", "o"]}}}]
+
+- ExpressionIndexOfCP:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: ExpressionIndexOfCP
+      active: [20]
+      operationMetricsName: AggExpressionIndexOfCP
+      pipeline: [{$project: {indexOfCP: { $indexOfCP: ["$hello", "l"]}}}]
+
+- ExpressionIsNumber:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: ExpressionIsNumber
+      active: [21]
+      operationMetricsName: AggExpressionIsNumber
+      pipeline: [{$project: {isNumber: { $isNumber: "$randomType"}}}]
+
+- ExpressionLet:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: ExpressionLet
+      active: [22]
+      operationMetricsName: AggExpressionLet
+      pipeline: [{$project: {let: { $let: {vars: { a: "$int", b: "$int2"}, in: {$gte: ["$$a", "$$b"]}}}}}]
+
+- ExpressionLn:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: ExpressionLn
+      active: [23]
+      operationMetricsName: AggExpressionLn
+      pipeline: [{$project: {ln: { $ln: "$posInt"}}}]
+
+- ExpressionLog10:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: ExpressionLog10
+      active: [24]
+      operationMetricsName: AggExpressionLog10
+      pipeline: [{$project: {log10: { $log10: "$posInt"}}}]
+
+- ExpressionMod:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: ExpressionMod
+      active: [25]
+      operationMetricsName: AggExpressionMod
+      pipeline: [{$project: {mod: { $mod: ["$int", "$posInt"]}}}]
+
+- ExpressionMultiply:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: ExpressionMultiply
+      active: [26]
+      operationMetricsName: AggExpressionMultiply
+      pipeline: [{$project: {multiply: { $multiply: ["$int", "$int2"]}}}]
```

Comment:
I have mixed feelings about keeping this. On one hand, we have coverage [here](https://github.com/mongodb/genny/blob/10ba24b5f9044092b8760c2bfaec37afee5b8e2d/src/phases/execution/RunLargeArithmeticOp.yml#L20) and [here](https://github.com/mongodb/genny/commit/3328524bc551fa3a5fbe37df21371c2a5e5517b6#diff-34fd154e6e541e2ae642757c7863eae09f9087adad440ea9aae349a942d279b3R81-R107) but these tests are about constant folding 100s of args and may not be representative of how users use $multiply in general. What do you think?

Reply:
Since they use constant folding, we can keep the test (at least till we implement the variadic expressions)

Reply:
Sounds good to me!